go.mod

```diff
@@ -184,3 +184,5 @@ replace k8s.io/metrics => k8s.io/metrics v0.0.0-20181128195641-3954d62a524d
 replace k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190122181752-bebe27e40fb7
 
 replace k8s.io/client-go => k8s.io/client-go v2.0.0-alpha.0.0.20190115164855-701b91367003+incompatible
+
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
This is the exact version as the one found in `go.sum` for thrift, so no production code should be changed.

https://github.com/jenkins-x/jx/pull/3321/files#diff-f949e2d81c8076ebbf8af38fcbb72c1fL14